src/sidebar/services/test/frame-sync-test.js

```diff
@@ -91,24 +87,27 @@ describe('sidebar.frame-sync', function() {
       this.startDiscovery = sinon.stub();
     }
 
-    angular.mock.module('app', {
-      store: fakeStore,
-      bridge: fakeBridge,
-    });
-
-    angular.mock.inject(function(_$rootScope_, _frameSync_) {
-      $rootScope = _$rootScope_;
-      frameSync = _frameSync_;
-    });
-
     $imports.$mock({
       '../../shared/discovery': FakeDiscovery,
     });
+
+    $rootScope = {
+      $broadcast: sinon.stub(),
+    };
+
+    frameSync = new Injector()
+      .register('$rootScope', { value: $rootScope })
+      .register('$window', { value: {} })
```

Comment:
nb. If one of these `.register` lines is accidentally omitted, the test runner surprisingly fails with output similar to https://hypothes-is.slack.com/archives/C1M8NH76X/p1585317200008900.

Reply:
Is this still the case after the `mocha` downgrade (to address exceptions in `beforeEach`)?

Reply:
This should be resolved after the mocha downgrade, although the `$window` dependency is still required.